lib/debugger/widgets/selectionList.lua

```diff
@@ -125,7 +158,9 @@ return function(Plasma)
 		local selected
 
 		for _, item in items do
-			if Item(item.text, item.selected, item.icon, item.sideText, options.width):clicked() then
+			if
+				Item(item.text, item.selected, item.icon, item.sideText, options.width, item.barWidth, item.index):clicked()
```

Comment:
should we remove options.width here since it appears unused?

Reply:
We could be here all day cleaning up the debugger code, but I would like to retain the scope of the PR

Reply:
It feels within scope since you are adding new arguments

Reply:
My previous question was an honest one not being familiar with plasma, but the `_` argument up top clued me in. If it is true we dont use this argument, I can make the commit for you and clean as we go

Reply:
It is used by `worldInspect.lua`, whether it is useful or not

Reply:
https://github.com/matter-ecs/matter/pull/35